### PR TITLE
doc: edit PendingReleaseNotes <v18 for Reef

### PR DESCRIPTION
**Pull request by zdover23**
Line edit PendingReleaseNotes for Reef.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: PendingReleaseNotes [hunk 1]
```diff
@@ -166,57 +166,59 @@
...
+  CephFS features. Any side effects experienced in previous Quincy versions
+  will no longer occur, provided that the configuration remains set to
+  ``false``. Relevant tracker: https://tracker.ceph.com/issues/55521
 
-* telemetry: Added new Rook metrics to the 'basic' channel to report Rook's
-  version, Kubernetes version, node metrics, etc.
-  See a sample report with `ceph telemetry preview`.
-  Opt-in with `ceph telemetry on`.
+* telemetry: Added new Rook metrics to the ``basic`` channel to report Rook's
+  version, the Kubernetes version, node metrics, and other information.  To see
```
Comment (colemitchell) [inline PendingReleaseNotes:178]: @zdover23 - A double space seems to have been introduced during the line wrap procedure.

File: PendingReleaseNotes [hunk 2]
```diff
@@ -166,57 +166,59 @@
...
+  a sample report, run ``ceph telemetry preview``.  To opt in, run ``ceph
```
Comment (colemitchell) [inline PendingReleaseNotes:179]: @zdover23 - Ditto.

File: PendingReleaseNotes [hunk 3]
```diff
@@ -166,57 +166,59 @@
...
+  https://tracker.ceph.com/issues/56147
+
+* The ``cephadm`` utility may now be configured to carry out CephFS MDS
+  upgrades without reducing ``max_mds`` to ``1``. Previously, Cephadm would
+  reduce ``max_mds`` to ``1`` in order to avoid certain potential
+  incompatibilities (for example: having two active MDS daemons modifying
+  on-disk structures with new versions, communicating
+  cross-version-incompatible messages). This action could be disruptive for
+  large-scale CephFS deployments because it is not easy for the cluster to
+  reduce active MDS daemons to ``1``.  NOTE: In order to take advantage of this
```
Comment (colemitchell) [inline PendingReleaseNotes:207]: @zdover23 - Ditto.

File: PendingReleaseNotes [hunk 4]
```diff
@@ -166,57 +166,59 @@
...
-* New MDSMap field `max_xattr_size` which can be set using the `fs set` command.
-  This MDSMap field allows to configure the maximum size allowed for the full
-  key/value set for a filesystem extended attributes.  It effectively replaces
-  the old per-MDS `max_xattr_pairs_size` setting, which is now dropped.
-  Relevant tracker: https://tracker.ceph.com/issues/55725
+* A new MDSMap field ``max_xattr_size`` has been introduced. It is set by 
+  running the ``fs set`` command. This MDSMap field is used to specify the
+  maximum size allowed for the full key-value set of a filesystem's extended
+  attributes. It replaces the existing per-MDS ``max_xattr_pairs_size``
+  setting, which has been eliminated.  Relevant tracker:
```
Comment (colemitchell) [inline PendingReleaseNotes:223]: @zdover23 - Ditto. Also, the pattern for the rest of the document is that _Relevant tracker: [link]_ begins on a new line. But I have no idea whether that's something worth preserving.

**Comment by zdover23** [comment]
Superseded by https://github.com/ceph/ceph/pull/52490.

**Pull request closed without merging**